lint/linter.py

```diff
@@ -976,6 +976,9 @@ def reposition_match(self, line, col, m, vv):
     @staticmethod
     def strip_quotes(text):
         """Return text stripped of enclosing single/double quotes."""
+        if len(text) < 3:
+            return text
```

Comment:
So, if the string is *just* the two quotes, you opt to return the two quotes. Makes sense to me.

Reply:
I thought about it that way: If you quote *something*, the thing must be at least 3 chars long. A typical edge case here.